buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -676,6 +676,10 @@ def set_test_misc() {
         TESTS.each { id, target ->
             target['extraTestLabels'] = buildspec.getVectorField("extra_test_labels", target).join('&&') ?: ''
         }
+    } else {
+        TESTS.each { id, target ->
+            target['extraTestLabels'] = ''
+        }
     }
```

Comment:
> // Only add extra test labels if the user has not specified a specific TEST_NODE

This comment would not seem to apply anymore then correct?  Even if that is the case, if a user wanted to target a specific TEST_NODE (machine) why would added extra test labels matter?

Reply:
Comment still applies. If they request a specific node, we don't want to be adding on extra labels. This could cause problems if the extra labels don't match the node they've requested (which is probably an edge case).